### PR TITLE
Added exported variable in main and amazon manifest files

### PR DESCRIPTION
**Pull request by SergiOnGit**

### CODE REVIEW

File: android/src/amazon/AndroidManifest.xml [hunk 1]
```diff
@@ -1,13 +1,12 @@
-
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
           package="com.dooboolab.RNIap">
-<application>
+<application android:exported="true">
```
Comment (hyochan) [inline android/src/amazon/AndroidManifest.xml:3]: @SergiOnGit Could you kindly explain how this fixes the compatibility issue? As far as I know this just allows another app to start the activity.

Comment (SergiOnGit) [reply android/src/amazon/AndroidManifest.xml:3]: @hyochan Sorry, looks like I wrote another library's PR title here.

Comment (SergiOnGit) [reply android/src/amazon/AndroidManifest.xml:3]: @hyochan Can you accept this PR? This is really important.

Comment (hyochan) [reply android/src/amazon/AndroidManifest.xml:3]: I am sorry for the late review. @SergiOnGit Could you kindly explain how this is important in `amazon` IAP?

Comment (Dallas62) [reply android/src/amazon/AndroidManifest.xml:3]: @hyochan This is blocking build on Android, don't if should be exported true or false, but it's blocking. #1571 

```bash
> Task :react-native-iap:processAmazonReleaseManifest FAILED
/Users/boristacyniak/Projects/Mobile/Monstera/node_modules/react-native-iap/android/src/main/AndroidManifest.xml Error:
        android:exported needs to be explicitly specified for <receiver>. Apps targeting Android 12 and higher are required to specify an explicit value for `android:exported` when the corresponding component has an intent filter defined. See https://developer.android.com/guide/topics/manifest/activity-element#exported for details.

```

Comment (SergiOnGit) [reply android/src/amazon/AndroidManifest.xml:3]: > @SergiOnGit Could you kindly explain how this fixes the compatibility issue? As far as I know this just allows another app to start the activity.

I don't know native code, I just know that without this app doesn't build. Tell you the truth it is very bad that this kind of good module have this bug for this long. This doesn't happen on windows only, I have tested on mac too and it's the same.

**Comment by wandersonalwes** [comment]
Please approve.

**Comment by wandersonalwes** [comment]
Without that, I'm having conflict to compile an app after updating the `buildscript`.

```
buildscript {
    ext {
        buildToolsVersion = "30.0.2"
        minSdkVersion = 23
        compileSdkVersion = 31
        targetSdkVersion = 31
        ndkVersion = "21.4.7075529"
        androidXAnnotation = "1.1.0"
        androidXBrowser = "1.0.0"
    }
}
```

**Review by hyochan — APPROVED** [review]
Sorry for the late review~!

I think there is enough detail to be merged.

Thanks for the fixes 🙏

**Pull request merged**